pipeline.config

```diff
@@ -0,0 +1,58 @@
+#
+# (C) Copyright IBM Corp. 2021, 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+INSIGHTS:
+  ID: "9b253889-dee0-47f4-9ea0-402306239aaa"
+CI:
+  DOCKER_IMAGE_NAME: "cohorting-app"
+  UMBRELLA_REPO_PATH: "https://github.ibm.com/watson-health-cohorting/wh-cohorting-umbrella.git"
```

Comment:
should this be pointing to a publicly accessible repo?

Reply:
No, we had discussed in a meeting that we wanted to point to our existing internal umbrella repo that was also being used for our FHIR server.